resources/certificate.rb

```diff
@@ -1,26 +1,31 @@
 unified_mode true
 
 provides :certificate
+default_action :install
 
 property :certfile, String
-property :cert_password, String
-property :keychain, String
-property :apps, Array
-
-action_class do
-  def keychain
-    new_resource.property_is_set?(:keychain) ? new_resource.keychain : ''
-  end
-end
+property :cert_password, String, sensitive: true
+property :keychain, String, required: true
```

Comment:
We changed the property in the `keychain` resource to `kc_file`, but we're keeping leaving it as `keychain` for the certificate resource? Personally, I think it would be better to have the properties align if they're referring to the same kind of file. Particularly important since it accepts `kc_passwd`.

And while we're on the topic, is there a reason we truncated these properties? `keychain_file` and `keychain_password` would be more preferable all across the board IMHO, for readability and clarity.

Reply:
Updating in `feature/keychain`.